db/migrate/20210411233938_add_reminder_type_to_shows.rb

```diff
@@ -0,0 +1,5 @@
+class AddReminderTypeToShows < ActiveRecord::Migration
+  def change
+  	add_column :shows, :reminder_type, :string, :null => false, :default => "Never"
```

Comment:
this is a nitpick, but since you went out of your way (correctly) to define the list of possible reminder types as a class constant, why not use it here?  ie `:default => Show::REMINDERS.first`

Reply:
Great idea, I will make that change. It also seems more stylish.